xbmc/pvr/windows/GUIWindowPVRChannels.cpp

```diff
@@ -124,9 +124,11 @@ bool CGUIWindowPVRChannels::Update(const std::string &strDirectory, bool updateF
   bool bReturn = CGUIWindowPVRBase::Update(strDirectory);
 
   /* empty list for hidden channels */
-  if (m_vecItems->Size() == 0 && m_bShowHiddenChannels)
```

Comment:
Is needed to go back from this to the normal view of channels, the view button is not usable anymore if nothing is in list.